src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -152,11 +181,39 @@ public final void refreshItems() {
 
     }
 
+    private int getClosedIssueSize() {
+        int count = 0;
+        for (GuiElement element: getElementsList()){
+            if (!element.getIssue().isOpen()){
+                count++;
+            }
+        }
+        return count;
+    }
```

Comment:
If you are familiar with Java 8's functional stuffs, you can do this in a more functional way like this:
```java
return getElementsList().stream().filter((element) -> !element.getIssue().isOpen()).count();
```